sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java

```diff
@@ -103,19 +103,19 @@ public void analyse(Project project, SensorContext context) {
     try {
       List<File> reports = getReports(settings, fs.baseDir(), reportPathKey());
       violationsCount = 0;
-
+      
       for (File report : reports) {
         CxxUtils.LOG.info("Processing report '{}'", report);
         try {
-          int prevViolationsCount = violationsCount;
```

Comment:
dont know if showing issues for each report is usefull, i get tons of messages because we have loads of reports. changed it to print only the summary of all the issues